### DISCUSSION TITLE
consult-rg wiki advice

### DISCUSSION THREAD

**Discussion by vitek** [minad/consult#974] [Q&A]
Hi! I had a problem with ripgrep snippet in the wiki:

https://github.com/minad/consult/wiki#restart-consult-ripgrep-in-parent-directory

it works only the first time, then it's doesn't because this-command was updated by other command. I've managed to fix this with the following code, but it doesn't go up it restarts rg with directory prompt:

https://github.com/vitek/configs/blob/master/emacs/site-lisp/vitja-vertico.el#L113

That would be great if you update wiki snippet.

**Comment by minad** [comment]
Hi, I don't actively maintain user contributed snippets in the wiki. If you have an improvement, feel free to update the wiki.
